dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A powerful data analysis library for Python"
+DESCRIPTION="Pandas is a Python package providing fast, flexible \
+and expressive data structures designed to make working with \
+\"relational\" or \"labeled\" data both easy and intuitive. \
+It aims to be the fundamental high-level building block for \
+doing practical, real world data analysis in Python."
+HOMEPAGE="https://pandas.pydata.org/"
+COPYRIGHT="2008-2012 AQR Capital Management, LLC
+	Lambda Foundry, Inc.
+	PyData Development Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/3e/56/82c4d4c049294f87ebd05b65fdcbc9ed68bd23fb0a7e4469caf9a75d199f/pandas-0.23.3.tar.gz"
+CHECKSUM_SHA256="9cd3614b4e31a0889388ff1bd19ae857ad52658b33f776065793c293a29cf612"
+SOURCE_DIR="pandas-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
Checked building cython_python, that worked for gcc2, cython_python3 fails to build "cc1: Invalid option -Wno-unused-result" so this would have to be:
```diff
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
```

Reply:
> Checked building cython_python, that worked for gcc2, cython_python3 fails to build "cc1: Invalid option -Wno-unused-result" so this would have to be:
> `ARCHITECTURES="!x86_gcc2 x86 x86_64"`

@Begasus, I did not check this, so I might be wrong, but I'd imagine that the `cc1: Invalid option -Wno-unused-result` error would only happen with gcc2. Are you sure you tried building `cython_python3_x86`? (You might have tried building `cython_python3` instead of `cython_python3_x86`.)
(If you did then I'd say something is probably wrong in the recipe.)

(BTW, in an ideal world, if I'm not mistaken, `cython_python` and `cython_python3` should be merged to a single recipe.)

Reply:
Oops, sorry @Begasus. I just understood what you meant.
You _already_ knew that  `cython_python3` does not build on x86_gcc2, so what you're saying is that, on x86_gcc2, pandas_x86 will need to use `cython_python3_x86`. (That's correct.)